config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+	namespace :api do
+    namespace :v1 do
+			namespace :gardens do
+				get '/:id/env_measurements', to: 'env_measurements#index'
```

Comment:
Ahh now I see why this got called an index. I'm wondering if this should be backed out of `api/v1/gardens/:id/env_measurements` as an `env_measurements` index and instead route to `api/v1/gardens/:id` as a `gardens` show?

I'm thinking `/gardens/:id` feels more RESTful and inherently one garden shows all of it's measurements.
Also wondering if we did keep `/gardens/:id/env_,easurements`, would we ever take it one more step to show a single env_measurement at a show `/gardens/:id/env_measurements/:id`?

Reply:
Good point.  I think breaking them out in to separate endpoints is a better long term strategy.  I am working on revising it now.  Thanks for your input

Reply:
I appreciate being able to talk through this together @Mycobee! Thanks for explaining your reasoning, I definitely see your point about namespacing this to an index the way you did instead of returning env_measurements back as part of a garden.

Continuing to think more about this, if we did change this we could make this a similar call to DarkSky's forecast endpoint where it's possible to limit the number of hour/day objects returned for a forecast, but that definitely increases the complexity so for now I like the path we're on.